### PR TITLE
Re-bundle karma test files when non-test JS files change.

### LINKED ISSUE #1623: Changing files in `src` does not correctly trigger new test runs
Tests run again, but the sources do not get rebundled.

Things work fine when changing files under `test`.

### PR DESCRIPTION
**Pull request by cramforce**
Not sure there is a better way to do this, but apparently karma only watches the test files themselves which is not really what one wants during test driven development or really ever.

Fixes #1623

### CODE REVIEW

File: test/functional/test-cid.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ import {timer} from '../../src/timer';
 import {installViewerService} from '../../src/service/viewer-impl';
 import * as sinon from 'sinon';
 
-describe('cid', () => {
+describe.only('cid', () => {
```
Comment (erwinmombay) [inline test/functional/test-cid.js]: remove?

**Comment by erwinmombay** [comment]
not on my work comp right now so cant test, but how is the update->test cycle with the change?

**Comment by cramforce** [comment]
@erwinmombay For me, updating a non-test JS file would without this change trigger a rerun on the tests, but it would not actually contain the changes. The rerun is caused by the gulp task rebuilding our dist directory which the tests only serve, but there is no rebuild.

With this change the difference is that on the next test run (that is automatically triggered) you actually do get your JS changes.

**Comment by jridgewell** [comment]
The issue is that we are [only including](https://github.com/ampproject/amphtml/blob/master/build-system/tasks/test.js#L88-L92) our test files with karma.

**Comment by cramforce** [comment]
But that is the right way to go it. I've seen lots of problems from just blindly including all these sources. If karma used `watchify` this problem would not exist. Would be certainly best to patch this upstream, but until then this is better.

**Comment by erwinmombay** [comment]
@cramforce LGTM

**Pull request merged**